website/src/pages/index.js

```diff
@@ -13,10 +13,10 @@ function HomepageHeader() {
     <header className={clsx('hero hero--primary', styles.heroBanner)}>
       <div className="container">
         <h1 className="hero__title">{siteConfig.title}</h1>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
+        <h2 className="hero__subtitle">{siteConfig.tagline}</h2>
```

Comment:
I disagree with changing `p` to `h2`.
See more: https://stackoverflow.com/a/3910470

Reply:
Correct! changed back to p - thanks!